### ISSUE TITLE
Question: Vulnerability Scann

### ISSUE THREAD

**Issue by majidmc2** [blackcrw/wprecon#12]
Hi, when you detect plugins, themes and etc, how do you detect a vulnerability on them? Do you use a Database of wordpress vulnerability?

**Comment by ffx64** [comment]
So, because we do not have access to the source code, it is quite complicated to explore to find a vuln ... (in this case, exploring manually).

In the case of wordpress not every vuln has a poc / exploit. But all the vulns that are reported are listed by wordpress itself or even by the vulns information sites.

Wprecon will be no different ... the vulnerabilities will be listed based on the plugins and their versions.

### Resume:
So in summary ... the vulnerabilities will be listed from the database.

### New:
A new version of wprecon that is about to be released will have a scripting system. "What would that be ?!" well, everyone could develop an exploit, a poc that can be used to exploit wordpress.
